Sources/Fuzzilli/Protobuf/operations.proto

```diff
@@ -130,6 +130,13 @@ message BeginArrowFunctionDefinition {
 message EndArrowFunctionDefinition {
 }
 
+message BeginAsyncArrowFunctionDefinition {
```

Comment:
Please put these below the `EndAsyncFunctionDefinition` message

Reply:
done